### PR TITLE
doc: add instructions to build for Windows on macOS

### PR DESCRIPTION
**Pull request by fortuna**

### CODE REVIEW

File: x/examples/fyne-tools/README.md [hunk 1]
```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
 Note: the generated APK is around 85MB.
 
 
+## Windows
+
+From macOS, you can build the app for Windows with MinGW.
+
+First install MinGW:
```
Comment (jyyi1) [inline x/examples/fyne-tools/README.md:23]: AFAIK, [Mingw-w64](https://www.mingw-w64.org/) is a fork of the original mingw (a.k.a., mingw32) and is maintained by different devs.

```suggestion
From macOS, you can build the app for Windows with Mingw-w64.

First install Mingw-w64:
```

File: x/examples/fyne-tools/README.md [hunk 2]
```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
...
+brew install mingw-w64
+```
+
+Build for 64-bit:
+
+```
+GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC="x86_64-w64-mingw32-gcc" go build .
+```
+
+32-bit:
```
Comment (jyyi1) [inline x/examples/fyne-tools/README.md:35]: I'm thinking whether we should still put 32-bit build instructions here, are there any real world PCs that are still 32 bit?

File: x/examples/fyne-tools/README.md [hunk 3]
```diff
@@ -14,6 +16,31 @@ go run fyne.io/fyne/v2/cmd/fyne package -os android && adb install Net_Tools.apk
 Note: the generated APK is around 85MB.
 
 
+## Windows
```
Comment (jyyi1) [inline x/examples/fyne-tools/README.md:19]: Maybe add "cross-compilation from macOS" to the title? Cuz Windows users can build it directly from Windows (might still need to install either `gcc` or `msvc`).

```suggestion
## Windows (x-compile)
```

**Review by jyyi1 — COMMENTED** [review]
Thanks. This is also how we can build Outline Client natively from macOS.

File: x/examples/fyne-tools/README.md (hunk 3 above)
Comment (fortuna) [reply x/examples/fyne-tools/README.md:19]: Updated the text.

File: x/examples/fyne-tools/README.md (hunk 1 above)
Comment (fortuna) [reply x/examples/fyne-tools/README.md]: Updated.

File: x/examples/fyne-tools/README.md (hunk 2 above)
Comment (fortuna) [reply x/examples/fyne-tools/README.md]: Agree. removed the 32-bit instructions.

**Review by jyyi1 — APPROVED** [review]
LGTM!

**Pull request merged**